app/views/_layout.njk

```diff
@@ -3,16 +3,16 @@
 {% block head %}
   <!--[if !IE 8]><!-->
   
-  <link href="/static/css/core.5a92f36305e7f24e04e1.css" rel="stylesheet">
+  <link href="/static/css/core.b4df847f49f547c4ea33.css" rel="stylesheet">
```

Comment:
This file should really be excluded from check-in (since it's the template that is the important file - this is just auto-generated)